include/aquamarine/backend/DRM.hpp

```diff
@@ -315,17 +319,29 @@ namespace Aquamarine {
                 uint32_t vrr_capable;  // not guaranteed to exist
                 uint32_t subconnector; // not guaranteed to exist
                 uint32_t non_desktop;
-                uint32_t panel_orientation; // not guaranteed to exist
-                uint32_t content_type;      // not guaranteed to exist
-                uint32_t max_bpc;           // not guaranteed to exist
+                uint32_t panel_orientation;   // not guaranteed to exist
+                uint32_t content_type;        // not guaranteed to exist
+                uint32_t max_bpc;             // not guaranteed to exist
+                uint32_t Colorspace;          // not guaranteed to exist
```

Comment:
accidental upper case?

Reply:
The same casing as the drm property name https://drmdb.emersion.fr/properties/3233857728/Colorspace
Doesn't really matter but feels reasonable to follow their casing just in case they are going to add `colorspace` and `COLORSPACE` %)

Reply:
eh, fine.